samples/net/sockets/dumb_http_server/src/response_big.html.bin

```diff
@@ -0,0 +1,38 @@
+HTTP/1.0 200 OK
+Content-Type: text/html; charset=utf-8
+
+<html>
+<style>

```

Comment:
Technically this is malformed HTML.  Need to add a <head> section (where the <style> would go).
Also, you don't talk about this "big" response in the documentation.  Looks like you could tweak the code to get this big response if you wanted to...

Reply:
> Technically this is malformed HTML.

Depends on which HTML ;-). For pre-5, based on XML with omitted and implied tags, it may be even valid, and browsers definitely can deal with it. Happy to add &lt;head> there of course. The point - if you keep looking, you may find more artifacts, but it was tested to work with the browsers...